cfme/tests/infrastructure/test_ssa_vddk.py

```diff
@@ -0,0 +1,84 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.common.vm import VM
+from cfme.configure.configuration.analysis_profile import AnalysisProfile
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.utils import conf
+from cfme.utils.hosts import setup_host_creds
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from markers.env_markers.provider import ONE_PER_VERSION
+
+pytestmark = [
+    pytest.mark.tier(3),
+    test_requirements.smartstate,
+    pytest.mark.meta(server_roles="+smartproxy +smartstate"),
+    pytest.mark.provider([VMwareProvider],
+                         selector=ONE_PER_VERSION, scope='module'),
+    pytest.mark.usefixtures('setup_provider_modscope')
+]
+
+vddk_versions = [
+    ('v5_5'),
+    ('v6_0'),
+    ('v6_5')
+]
+
+
+@pytest.fixture(scope="module")
+def ssa_analysis_profile():
+    collected_files = []
+    for file in ["/etc/hosts", "/etc/passwd"]:
+        collected_files.append({"Name": file, "Collect Contents?": True})
+
+    analysis_profile_name = 'ssa_analysis_{}'.format(fauxfactory.gen_alphanumeric())
+    analysis_profile = AnalysisProfile(name=analysis_profile_name,
+                                       description=analysis_profile_name,
+                                       profile_type=AnalysisProfile.VM_TYPE,
+                                       categories=["System"],
+                                       files=collected_files)
```

Comment:
REQUIRED: please follow our [style guide](http://cfme-tests.readthedocs.io/guides/dev_guide.html) and use one indent.